gui/src/app/components/TabWidget.tsx

```diff
@@ -56,7 +73,9 @@ const CustomTabPanel = (props: TabPanelProps) => {
       style={{ height: "100%", width: "100%" }}
       {...other}
     >
-      {value === index && <>{children}</>}
+      {(value === index || tabsThatHaveBeenViewed.includes(index)) && (
```

Comment:
I don’t believe there is any circumstance where the first condition is true and the second is false, so this could be simplified to just `mounted &&` with the above change